docs/products/postgresql/concepts/high-availability.md

```diff
@@ -1,116 +1,121 @@
 ---
-title: High availability
+title: High availability of Aiven for PostgreSQL®
```

Comment:
As most (nearly all?) of the info on this page is duplicated in the general HA article, I wonder if it'd be better to just remove this and redirect there? The primary/standby node definitions in the terminology could also be added to the HA article instead of living in a separate page?

Reply:
Although indeed a huge part of this content can be shared by PG and MySQL, there are some technical details that actually differ these two articles, which is why reusing this content or cross-linking is not possible. Also, people are specifically requesting this type of info both for PG and MySQL so creating two separate articles seems the only way to address that temporarily before we have some better info architectural solutions for similar cases.